robot/EDA/resources/EDA.py

```diff
@@ -344,6 +344,11 @@ def verify_app_exists(self, app):
         locator = eda_lex_locators["app_tile"].format(app)
         self.selenium.wait_until_page_contains_element(locator, timeout=60, error=f'{app} did not open in 1 min')
 
+    def verify_item_exists(self, item):
```

Comment:
@rhakeshj since this is for the App Launcher, would it be better to call this `verify_app_exists`

Reply:
In salesforce it is displayed under all items (tabs) after clicking view all so went with the same language